docs/assets/scripts/docs.js

```diff
@@ -1,4 +1,38 @@
 $(function() {
+  function filterOrderSideNavItem(pageUrl, items) {
+    var ORDER = [
+      'ShopifyBuy',
+      'ShopClient',
+      'ProductModel',
+      'ProductVariantModel',
+      'ProductOptionModel',
+      'CartModel',
+      'CartLineItemModel'
+    ];
+
+    var newItems = ORDER.map(function(key) {
+      return items.find(function(docClass) {
+        return docClass.name === key;
+      });
+    })
```

Comment:
This should be fixed. https://github.com/Shopify/js-buy-sdk/pull/233/files#diff-e5d789301f1d0e399ded55169fc3ccf0R13